wsgi_kerberos.py

```diff
@@ -202,4 +209,8 @@ def custom_start_response(status, headers, exc_info=None):
             return self._unauthorized(start_response, server_token)
```

Comment:
Should this use the new `extra_body` param to include some "missing user" message in the 401 body?

Reply:
Done in the latest revision.